templates/base.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+    {% block title %}
+      <title>Taxi Service</title>
+    {% endblock %}
+  <title>Title</title>
+  {% load static %}
+  <link rel="stylesheet" href="{% static 'css/style.css'%}">
+</head>
+<body>
+  <div class="sidebar">
+    {% block sidebar %}
+      {% include "includes/sidebar.html" %}
+    {% endblock %}
+  </div>
+  <div class="content">
+    {% block content %}
+    {% endblock %}
+  </div>
```

Comment:
You have to create sidebar on pages
Use these classes for divs:
```
<div class="container-fluid">
     <div class="row">
        <div class="col-sm-3">
            (for sidebar)
        </div>
        <div class="col-sm-9">
            (for content)
        </div>
     </div>
</div>
```

Reply:
I looked through the README several times and didn't find a word about it, but OK)

Reply:
![image](https://github.com/user-attachments/assets/939c296c-e728-4e5c-af48-5caeb2a2a127)